doc/kubernetes/modules/ROOT/pages/load-behavior.adoc

```diff
@@ -0,0 +1,47 @@
+= Keycloak under load
```

Comment:
@ahus1 I think this makes sense. We should def follow up with another task, to add pages such as., Load Characteristics (which defines a certain load condition) and Benchmark Numbers (which captures the real world behavior of the application) which attests to the behavior we explained in this doc.

Reply:
+1 for this. It could use some of the baseline scenarios we ran; including the one we recently ran around logins. I think it would be a good fit in that category.